excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/RoleToAccessProfilesParser.java

```diff
@@ -0,0 +1,84 @@
+                                                  ParseContext parseContext) {
+        try {
+            final List<DefinitionDataItem> roleToAccessProfiles = definitionSheets
+                .get(SheetName.ROLE_TO_ACCESS_PROFILES.getName()).getDataItems();
+            return roleToAccessProfiles
+                .stream()
+                .map(questionItem ->
+                    createRoleToAccessProfileEntity(parseContext, questionItem)).collect(Collectors.toList());
+        } catch (InvalidImportException invalidImportException) {
+            ValidationResult validationResult = new ValidationResult();
+            validationResult.addError(new ValidationError(invalidImportException.getMessage()) {
+                @Override
+                public String toString() {
+                    return getDefaultMessage();
+                }
+            });
+            throw new ValidationException(validationResult);
+        }
+    }
+
+    public RoleToAccessProfileEntity createRoleToAccessProfileEntity(ParseContext parseContext,
+                                                                     DefinitionDataItem definitionDataItem) {
+        RoleToAccessProfileEntity roleToAccessProfileEntity = new RoleToAccessProfileEntity();
+
+        final String caseType = definitionDataItem.getString(ColumnName.CASE_TYPE_ID);
+        CaseTypeEntity caseTypeEntity = parseContext.getCaseTypes()
+            .stream()
+            .filter(entity -> entity.getReference().equals(caseType))
+            .findAny()
+            .orElseThrow(() -> new InvalidImportException("Case Type not found " + caseType));
+        roleToAccessProfileEntity.setCaseType(caseTypeEntity);
+
+        roleToAccessProfileEntity.setRoleName(definitionDataItem.getString(ColumnName.ROLE_NAME));
+        roleToAccessProfileEntity.setLiveFrom(definitionDataItem.getDate(ColumnName.LIVE_FROM));
+        roleToAccessProfileEntity.setLiveTo(definitionDataItem.getDate(ColumnName.LIVE_TO));
+        roleToAccessProfileEntity.setReadOnly(definitionDataItem.getBoolean(ColumnName.READ_ONLY));
+        roleToAccessProfileEntity.setDisabled(definitionDataItem.getBoolean(ColumnName.DISABLED));
+        String accessProfiles = definitionDataItem.getString(ColumnName.ACCESS_PROFILES);
+        validateAccessProfiles(caseType, accessProfiles, parseContext);
+        roleToAccessProfileEntity.setAccessProfiles(accessProfiles);
+        roleToAccessProfileEntity.setAuthorisation(definitionDataItem.getString(ColumnName.AUTHORISATION));
+
+        return roleToAccessProfileEntity;
+    }
+
+    private void validateAccessProfiles(String caseType,
+                                        String accessProfiles,
+                                        ParseContext parseContext) {
+        if (StringUtils.isEmpty(accessProfiles)) {
+            throw new InvalidImportException("Access Profiles should not be null or empty");
+        }
+
+        Arrays.stream(accessProfiles.split(","))
```

Comment:
Is this a good place for a validation? Aren't we supposed to parse the definition and then validate the model?

Reply:
that's a good point, will have a look into this

Reply:
Added new validator